packages/e2e-tests/specs/experiments/blocks/comments-query.test.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	activateTheme,
+	clickBlockAppender,
+	createNewPost,
+	getEditedPostContent,
+	insertBlock,
+	setOption,
+} from '@wordpress/e2e-test-utils';
+
+describe( 'Comment Query Loop', () => {
+	beforeAll( async () => {
+		await activateTheme( 'emptytheme' );
+		await setOption( 'page_comments', true, 'options-discussion.php' );
+		await setOption( 'comments_per_page', '2', 'options-discussion.php' );
+		await createNewPost();
+	} );
+	it( 'Comment Query block insertion', async () => {
+		// We won't select the option that we updated and will also remove some
+		// _transient options that seem to change at every update.
+		await insertBlock( 'Comments Query Loop' );
+		expect( await getEditedPostContent() ).toMatchSnapshot();
+	} );
+	it( 'can be created by typing "/comment query loop"', async () => {
+		// Create a list with the slash block shortcut.
+		await clickBlockAppender();
+		await page.keyboard.type( '/comments query loop' );
+		await page.waitForXPath(
+			`//*[contains(@class, "components-autocomplete__result") and contains(@class, "is-selected") and contains(text(), 'Comments Query Loop')]`
+		);
+		await page.keyboard.press( 'Enter' );
+		expect( await getEditedPostContent() ).toMatchSnapshot();
+	} );
```

Comment:
I'm afraid the [same arguments](https://github.com/WordPress/gutenberg/pull/39502/files#r830318752) as above apply 😕  It's a rather generic test that covers pretty standard block behavior -- the only reason it would fail that would be more or less specific to the Query Loop block is if it's not registered or otherwise broken. But as said above, we should cover block insertion as part of a test that covers a behavior that's more specific to the individual block (and we don't really need to cover different ways to insert the same block).

If we currently don't have any more block-specific tests (cause they're e.g. already covered by the unit test), I'd go as far as to suggest to remove them 😅